### DISCUSSION TITLE
Drag div border to resize

### DISCUSSION THREAD

**Discussion by devstc** [mantinedev/mantine#1691] [Q&A] [answered]
I'm a Mantine noobie, just started working with it yesterday.  Does Mantine provide any way to drag a vertical border between divs to resize them?  If not, does anyone know a Mantine-friendly solution that could work?  I've installed and tried a couple things... 'react-resize-panel' and 'allotment'.  I couldn't get either to do what I wanted.  And Google hasn't turned up as much as I would have thought I would find for a solution on this.  I didn't find anything in the discussions here either.  Any help is appreciated.  Thanks in advance!

**Comment by rtivital** [comment, marked as answer]
Mantine does not provide components that handle resizing, you should use a third-party library for that – any will work well with Mantine. I cannot recommend any library in particular, as I haven't used any of them.

**Comment by devstc** [comment]
Vitaly... Thanks for the response.  I'll keep checking some other possibilities.  I understand if Mantine doesn't have that feature, but I honestly thought I would find plenty of options out there for such functionality.  I'll keep looking.  And thanks for what looks to be such a great library!  I did quite a bit of researching and reading, even passing on the big, tried-and-true names.  Mantine had a couple things that won me over... drag-and-drop, the text editor, and a dark theme to name a few.  Plus people seemed to have a lot of good things to say about it.  Appreciate all the hard work!  Keep innovating!
